build/download-manager.js

```diff
@@ -98,7 +98,15 @@ var DownloadManager = {
                    .createInstance(Ci.nsIDownloader);
     downloader.init(listener, tmpFile);
 
-    let channel = Services.io.newChannel(url, null, null);
+    let channel = Services.io.newChannel2(url,
+                                          null,
+                                          null,
+                                          null,      // aLoadingNode
+                                          Services.scriptSecurityManager.getSystemPrincipal(),
```

Comment:
This line is throwing a JSHint error. You will need to fix the length, probably by changing the indentation.

build/download-manager.js: line 105, col 94, Line is too long. (ERROR)